index.html

```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link rel="stylesheet"  type="text/css" href="application.css">
+  </head>
+  <body>
+    <div class="book-detail">
+      <h1 class="title">Título del libro <span class="placeholder genre">(género)</span></h1>
+      <div class="description">
+        <h4 class="subtitle">Autor del libro:</h4><p class="placeholder">Nombre del autor del libro</p>
```

Comment:
Justo a comment, put 'p' in a new line. For this you can use a html formatter plugin in your IDE